src/scval.js

```diff
@@ -91,6 +91,7 @@ import { ScInt, XdrLargeInt, scValToBigInt } from './numbers/index';
  * nativeToScVal(true); // scvBool
  * nativeToScVal([1, 2, 3]);                    // gives scvVec with each element as scvU64
  * nativeToScVal([1, 2, 3], { type: 'i128' });  // scvVec<scvI128>
+ * nativeToScVal([1, '2'], { type: ['i128', 'symbol'] });  // scvVec with diff types
```

Comment:
might be worth highlighting one more example where the val array has more elements than type array, and how auto typing is applied on the additional val's?